public/util/data.ts

```diff
@@ -864,16 +864,30 @@ export function sortVariablesByImportance(variables: Variable[]): Variable[] {
       rankMap = resultsGetters.getFeatureImportanceRanking(store)[solutionId];
     }
   }
+  const findImageAndGeoTypes = (v: Variable) => {
```

Comment:
maybe this fn should be renamed `isImageOrGeoType` or something along those lines?